src/presentation/http/router/note.test.ts

```diff
@@ -518,6 +560,108 @@ describe('Note API', () => {
 
       expect(response?.json().message).toStrictEqual(expectedMessage);
     });
+
+    test.each([
+      /** Returns two parents in case of relation between child and parent notes with 200 status */
+      {
+        numberOfNotes: 2,
+        userNoteCreationDifferent: false,
+        isPublic: true,
+      },
```

Comment:
add expectedStatusCode and cover cases, when user is not authorized or has no access to note